content/en/news/release-notes.md

```diff
@@ -6,6 +6,37 @@ weight: 1
 
 For additional information check our [sprint demo videos](https://www.youtube.com/channel/UCcm2NzDN_UCZKk2yYmOpc5w) and [blogs](https://medium.com/kialiproject).
 
+## 1.52.0
+Sprint Release: June 10th, 2022
+
+Features:
+
+* [Define a structure for future plugins development](https://github.com/kiali/kiali/issues/4934)
+* [Update beta interfaces for CronJob workloads](https://github.com/kiali/kiali/issues/4519)
+* [Adjust font style in charts options](https://github.com/kiali/kiali/issues/5168)
+* ["This subset's labels are not found in any matching host" - DestinationRule and ServiceEntry](https://github.com/kiali/kiali/issues/5131)
+* [plugin should log its version (git commit or some build version)](https://github.com/kiali/openshift-servicemesh-plugin/issues/16)
```

Comment:
this is related to the plugin - not related to the product associated with the release notes
```suggestion
```